coopersystem-fsd/pcc
# Change events order to ASC

## Description

Added order by start date in events

## Steps to test

1. Create a page and set the new template "Events"
2. Open the new page created
3. You should be able to see the events in start date order ASC

**Expected behavior:** Show events page order by start date
